internal/sourcecode/templates/redwood/Dockerfile

```diff
@@ -39,7 +39,7 @@ COPY --from=api_build /app/api/dist /app/api/dist
 COPY --from=api_build /app/api/db /app/api/db
 COPY --from=api_build /app/node_modules/.prisma /app/node_modules/.prisma
 
-EXPOSE 8911
+EXPOSE 8910
 
 # Entrypoint to @redwoodjs/api-server binary
-CMD [ "yarn", "rw-api-server", "--port", "8911", "--rootPath", "/api" ]
+CMD [ "yarn", "rw-server", "--port", "8910" ]
```

Comment:
Do we need any more CLI options for the rewrites to work?

Reply:
It should be able to figure out the root path from redwood.toml automatically:

https://github.com/redwoodjs/redwood/blob/902b36d4a3907e7e1a99930769c539b15d3c3b6f/packages/api-server/src/cliHandlers.ts#L89